site-src/contributing/docs.md

```diff
@@ -0,0 +1,140 @@
+# Contributing to the docs
```

Comment:
Added a contributors guide for the docs that outlines the versioning process as well as other things like local preview